### PR TITLE
Finish leaderboard Project

### PR DESCRIPTION
**Pull request by zunairkhan811**
The Project is completed and styling has been added to the project.

### CODE REVIEW

**Review by wayungi — COMMENTED** [review]
Hi @zunairkhan811 , 
image

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌ ❌ 

- [x] You have submitted a project, that has incomplete parts in the `readme`. Please thoroughly edit your `readme`
invalid


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -108,15 +108,6 @@ To run the project, execute the following command:
...
-
-### Run tests
-
-To run tests, run the following command:
-
-<!--
-In order to run for linter errors, you will have to create a pull request
---->
-
 ### Deployment
```
Comment (OLIPLICHE) [inline README.md]: - [x] Hi, I would advise you to Add in the Readme all that is needed for running the app. I am aware that this requirement was not mandatory. It is important to guide the user in how to Run your App it shows a lot of professionalism. 
- Here is what I would suggest: Under the section `Usage` in Readme, add  `npm install`, `npm run build`, and `npm run start`.

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @zunairkhan811, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- App is properly styled 👍 
- The layout looks like the wireframe 👍 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by medaminedev66** [comment]
Hello @zunairkhan811👋,

_This is an additional review as a result of the second opinion request._

The PR was supposed to be invalidated instead of requesting for additional instructions on how to launch the web app by the user. The CR should invalidate the PR if they believe that the Getting started part of the project does not have adequate instructions on how to use the project or if they are unable to execute the app.

Even so, as a completely optional suggestion, I still recommend you to include some instructions in your readme on how the user can execute the web application.

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approve right away :)

Keep rocking 🚀!

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @zunairkhan811 ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️
- Good commit messages ✔️
- Linters are passing ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**